include/ignition/physics/OperateOnSpecifiedData.hh

```diff
@@ -17,3 +17,8 @@
 
 #include <gz/physics/OperateOnSpecifiedData.hh>
 #include <ignition/physics/config.hh>
+
+#define IGN_PHYSICS_OPERATEONSPECIFIEDDATA_TEMPLATES \
+  GZ_PHYSICS_OPERATEONSPECIFIEDDATA_TEMPLATES
```

Comment:
These are defined in the `detail` directory, can we skip the tick-tock?

Reply:
https://github.com/gazebosim/gz-physics/pull/353/commits/c18e0f04d406ad066d91dbca97f2f4af63e16def :boom: